client/src/assets/scss/overrides.scss

```diff
@@ -1,30 +1,42 @@
 /*
- * Vuetify Overrides
+ * Vuetify overrides
  */
 
 @import "theme.scss";
 
 .v-application {
-  font-size: 1rem; /* 16px */
-  line-height: 1.5rem; /* 24px */
+  font-size: 1rem;
+  line-height: 1.5rem;
   font-family: 'BCSans', Verdana, Arial, sans-serif !important;
   padding: 0;
-}
-
-.v-application .text-body-1 {
-  font-family: 'BCSans', Verdana, Arial, sans-serif !important;
-  letter-spacing: normal !important;
-}
 
-.v-application .text-body-2 {
-  font-family: 'BCSans', Verdana, Arial, sans-serif !important;
-  letter-spacing: normal !important;
-  line-height: 1.375rem !important; /* 22px */
+  .text-body-1 {
+    font-family: inherit !important;
+    letter-spacing: normal !important;
+    line-height: 1.5rem !important;
+  }
+  
+  .text-body-2 {
+    font-family: inherit !important;
+    letter-spacing: normal !important;
+    line-height: 1.375rem !important;
+  }
+  
+  .text-name {
+    font-family: inherit;
+    font-size: 1.125rem;
+    line-height: 1.875rem;
+    text-transform: capitalize;
+  }
 }
 
 .v-btn {
-  letter-spacing: normal;
-  border-bottom-width: 1px !important;
+  font-family: inherit !important;
+  font-size: 0.875rem !important;
```

Comment:
This is the global font size for buttons per Tracey (with some special cases).

Although this is a global style, there are misc overrides for this in various scss/sass/vue files. Please clean those up if you're working in those files.

Reply:
Will definitely keep an eye on it after merge 👍

Reply:
Yeah, or you download my branch and have a look at the pages you're familiar with (please and thanks).

Reply:
Can do!